test/test.js

```diff
@@ -2472,6 +2472,64 @@ describe('Page', function() {
 
     }));
   });
+
+  describe('Target', function() {
+    it('Browser.targets should return all of the targets', SX(async function() {
+      // The pages will be the testing page and the original newtab page
+      const targets = browser.targets();
+      expect(targets.some(target => target.type() === 'page' &&
+        target.url() === 'about:blank')).toBeTruthy('Missing blank page');
+      expect(targets.some(target => target.type() === 'other' &&
+        target.url() === '')).toBeTruthy('Missing browser target');
```

Comment:
what's the argument in `toBeTruthy`?

Reply:
It is the error message when the condition fails.